### DISCUSSION TITLE
File block

### DISCUSSION THREAD

**Discussion by subhash16** [Sketchware-Pro/Sketchware-Pro#811] [Q&A]
File block is not working android 12.
Please fix this issue even I can't delete file from download folder but can write.

**Comment by j-px** [comment]
Sketchware's Blocks (the same ones are in use in Sketchware Pro) use `java.io.File`, and if they don't work for you, refer to #293.

**Comment by subhash16** [comment]
java.io.File I didn't understand this

**Comment by subhash16** [comment]
.

**Reply by j-px** [reply]
could you not spam useless comments please, or at least ask questions properly if you have them?

**Reply by subhash16** [reply]
I just closed this issue

**Reply by j-px** [reply]
I'm pretty sure you can do that without commenting

**Reply by subhash16** [reply]
Sorry for that .
